config.py

```diff
@@ -17,6 +17,10 @@ class Config(object):
     SURVEY_USERNAME = os.getenv('SURVEY_USERNAME')
     SURVEY_PASSWORD = os.getenv('SURVEY_PASSWORD')
     SURVEY_AUTH = (SURVEY_USERNAME, SURVEY_PASSWORD)
```

Comment:
Kind of picky but could add a couple of line spaces in this file to make it a little easier to read, maybe split up service specific configs?